### DISCUSSION TITLE
Posición de llegada de nuevos puffles

### DISCUSSION THREAD

**Discussion by diegoftpxd** [IIC2133-PUC/2024-2#16] [Tarea 0] [answered]
Hola, me quedan dudas con respecto a cuál es la posición que debe ocupar un puffle al ser agregado al array. Si tengo un pingüino que tiene varios puffles y compra uno nuevo, ¿dónde debe llegar ese puffle?

En el sentido de que, si tengo el siguiente array **[0,1,  ,3,  ,  ,]**, donde esos números son las IDs de los puffles, al momento de comprar un nuevo puffle, ¿este se debería ubicar en la posición 2 o en la 4?

En caso de ubicarse en la 4, si tengo el array **[0,1,  ,3,   ,5,6]** donde la última casilla está ocupada pero tengo casillas anteriores desocupadas, ¿tengo que ubicarlo en la casilla 2 o en la casilla 4

**Comment by sebacarrascop** [comment, marked as answer]
Hola, no podemos responder específicamente sobre el arreglo que mencionas, ya que eso depende de tus decisiones de diseño.

Con respecto a la posición dentro de un inventario, si un pingüino compra un nuevo puffle, la forma en que se almacene debe ser consistente con el orden en que fue añadido a este: el último puffle añadido debe estar en la última posición del inventario, a menos que se indique lo contrario, como en otros eventos donde los puffles deben ser colocados en posiciones específicas.

**Reply by diegoftpxd** [reply]
graciaas!
